admittance_controller/README.md

```diff
@@ -0,0 +1,146 @@
+admittance_controller
+==========================================
+
+The admittance controller operates in one of two modes: chained and non-chained. 
+In the non-chained mode, the controller listens to joint commands on the `~/joint_commands` topic to get a reference. In chained mode, the admittance controller receives its reference from another controller. In both modes, the controller applies the calculated admittance values to the received reference and write the modified reference to its command interfaces.     
+
+The controller requires an external kinematics plugin to function. The [kinematics_interface](https://github.com/ros-controls/kinematics_interface) repository provides an interface and an implementation that the admittance controller can use.
+
+## Parameters:
+admittance.damping_ratio
```

Comment:
I was thinking this could be auto-generated from the yaml so users could read it more easily. But for now, I just added it manually. I can remove if you think its not worth it. What do you think?

Reply:
Done